internal/service/streamprocessor/resource.go

```diff
@@ -0,0 +1,254 @@
+package streamprocessor
+
+import (
+	"context"
+	"errors"
+	"net/http"
+	"regexp"
+
+	"github.com/hashicorp/terraform-plugin-framework/path"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/config"
+	"go.mongodb.org/atlas-sdk/v20240805001/admin"
+)
+
+const streamProcessorName = "stream_processor"
```

Comment:
looks like we are defining this in two places (`data_source.go#StreamProcessorName`). Let's reuse it

Reply:
Done, data_source was using a wrong name, so using this one instead fixes the name. Good catch